src/contacts/view/ContactViewModel.ts

```diff
@@ -72,12 +72,10 @@ export class ContactViewModel {
 
 	private updateUrl() {
 		const contactId =
-			this.targetContactId != null
-				? this.targetContactId
-				: !this.listModel.state.inMultiselect && this.listModel.getSelectedAsArray().length === 1
+			!this.listModel.state.inMultiselect && this.listModel.getSelectedAsArray().length === 1
 				? getElementId(this.listModel.getSelectedAsArray()[0])
 				: null
-		if (contactId) {
+		if (contactId && !styles.isSingleColumnLayout()) {
```

Comment:
viewModel should not be aware of styles

Reply:
This is a good point. The proper alternative would be to have an additional parameter in `init()` to handle whether the contact should be remembered  or another function altogether. It is a refactor however so I choose this instead to avoid stepping on toes. I will push a commit with a refactor and it can be dropped if it is not preferable.

Reply:
I have pushed a refactored approach. It can be changed/dropped if needed.